config/clusters/meom-ige/common.values.yaml

```
@@ -92,7 +92,6 @@ basehub:
         - display_name: "Small"
           default: true
           allowed_teams: &allowed_teams_normal_use
-            - 2i2c-org:hub-access-for-2i2c-staff
             - meom-group:hub-users # long term users
```

Removing us from `allowed_teams` was a mistake, if this is defined and our github team isn't part of this, we won't be able to see this profile.

```suggestion
            - 2i2c-org:hub-access-for-2i2c-staff
            - meom-group:hub-users # long term users
```